### PR TITLE
Format preferences page

### PR DESCRIPTION
**Pull request by faraplay**
This pull request makes the preferences page narrower so that it fits on small screens.
Comparison:

Before | After
-------|--------
Preferences before | Preferences after

### CODE REVIEW

**Comment by faraplay** [comment]
It would probably be better to use a theme color for the border color to fit both light and dark color schemes

File: js/cs.js [hunk 1]
```diff
@@ -3,4 +3,5 @@ import "./general.js";
 import "./header.js";
 import "./sidebars.js";
 import "./posts";
+import "./preferences.js";
```
Comment (OctoNezd) [inline js/cs.js:6]: Please add same import in user.js

File: css/customUi.css [hunk 2]
```diff
@@ -38,3 +38,8 @@ form.usertext {
 #header {
     display: none;
 }
+
+div.pref-row {
+    border-top: 1px dotted gray;
```
Comment (OctoNezd) [inline css/customUi.css:43]: I assume the correct color would be var(--md-sys-color-outline-variant), per https://m3.material.io/components/divider/specs

**Comment by OctoNezd** [comment]
Please border color with suggested change. You can fake light mode by clicking on 
image
in devtools

**Pull request merged**